apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -285,7 +285,7 @@ const trackInnerBlocksReplacement = ( rootClientId, blocks ) => {
 		block_name: name,
 		blocks_replaced: true,
 		// isInsertingPageTemplate filter is set by Starter Page Templates
-		from_template_selector: applyFilters( 'isInsertingPageTemplate', false ),
+		from_template_selector: applyFilters( 'isInsertingPagePattern', false ),
```

Comment:
Let's keep the old hook took for backward compatibility. Let's be on the safe side 😄

Reply:
Yeah that makes sense 😁 .  Updated!